docs/tutorials/04_torch_qgan.ipynb

```diff
@@ -918,27 +717,15 @@
     }
    },
    "source": [
-    "On the plot above, in the <span style=\"color:blue\">blue color</span> the generated distribution is shown and\n",
-    "in the <span style=\"color:orange\">orange color</span> the training one. You may find that both CDFs are similar to each other."
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {
-    "pycharm": {
-     "name": "#%% md\n"
-    }
-   },
-   "source": [
-    "### Conclusion\n",
+    "## 7. Conclusion\n",
     "\n",
-    "Quantum generative adversarial networks employ the interplay of a generator and discriminator to map an approximate representation of a probability distribution underlying given data samples into a quantum channel. This tutorial presents a self-standing PyTorch-based qGAN implementation where the generator is given by a quantum channel, i.e., a variational quantum circuit, and the discriminator by a classical neural network, and discusses the application of efficient learning and loading of generic probability distributions – implicitly given by data samples – into quantum states.\n",
+    "Quantum generative adversarial networks employ the interplay of a generator and discriminator to map an approximate representation of a probability distribution underlying given data samples into a quantum channel. This tutorial presents a self-standing PyTorch-based qGAN implementation where the generator is given by a quantum channel, i.e., a variational quantum circuit, and the discriminator by a classical neural network, and discusses the application of efficient learning and loading of generic probability distributions into quantum states.\n",
     "Since, this approximate loading requires only $\\mathscr{O}\\left(poly\\left(n\\right)\\right)$ gates and can enable the use of potentially advantageous quantum algorithms by offering an efficient data loading scheme."
```

Comment:
I must admit I struggled a bit here coming into this last sentence from the prior. I am thinking the way I come into this last sentence is to focus of the bit before that says `efficient learning and loading` and the Since, here is justifying the use of efficient there.

Reply:
Ah, I must admit, I struggled as well. The text came from the original Christa's paper and was in the previous version as well. I adjusted it slightly, literally copied from the paper. May be it is better now. Feel free to suggest other edits. I don't really know how to improve.

Reply:
To clarify things a bit the issue I mainly had,  is that it starts out with `Since,...` and gives an explanation as is its justifying a claim made before. I cannot find a direct claim that say states that a Quantum QGAn (is/can be) more efficient than a classical equivalent. It says ` discusses the application of efficient learning a` and says efficient but without any explicit claim. Hence me saying above, I believed it was justifying the use of efficient in that text - but without any direct claim that the Since was leading into the explanation/why I struggled going into that sentence to really connect it.

Reply:
No, your update made things a lot better, thx.